client/extension.ts

```diff
@@ -58,13 +58,29 @@ export function activate(context: vscode.ExtensionContext): void {
         const uri = QueryResultsContentProvider.encodeLocation(editor.document.uri, host, port);
         editorSparqlQuery(client, actualQuery, uri, editor, provider);
     });
-    const sendRowsQuery = vscode.commands.registerTextEditorCommand('extension.sendRowsQuery', editor => {
+    const sendRowsJsonQuery = vscode.commands.registerTextEditorCommand('extension.sendRowsJsonQuery', editor => {
         const actualQuery: string = editor.document.getText();
         const cfg: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration();
         const client: MarklogicClient = cascadeOverrideClient('', SJS, cfg, context.globalState);
         const host = client.params.host; const port = client.params.port;
         const uri = QueryResultsContentProvider.encodeLocation(editor.document.uri, host, port);
-        editorRowsQuery(client, actualQuery, uri, editor, provider);
+        editorRowsQuery(client, actualQuery, uri, editor, provider, 'json');
+    });
+    const sendRowsCsvQuery = vscode.commands.registerTextEditorCommand('extension.sendRowsCsvQuery', editor => {
```

Comment:
To avoid the duplication, I think you can make a function that accepts the format arg and then just do something like:

    registerTextEditorCommand('...', editor => sendEditorRowsQuery(editor, "json"))

Where `sendEditorRowsQuery` would have these repeated 6 lines.